documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -1,54 +1,125 @@
+[//]: # (Start an issue with title "[WIP] Lotus Node|Miner vX.Y.Z" and adjust the title for whether it's a Node or Miner release.)
+[//]: # (Copy in the content of https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md)
+[//]: # (Find/Replace "X.Y.Z" with the actual values.)
+[//]: # (Copy/paste the "Release Checklist > RCX" section to "Release Checklist > Stable \(non-RC\) Release" and apply the "diff" called out there.)
+[//]: # (Find/Replace "RCX" with "RC1".)
+[//]: # (If this isn't a release tied to a network upgrade, remove all items with "\(network upgrade\)")
+[//]: # (Adjust the "Meta" section values.)
+[//]: # (Apply the `tpm` label to the issue)
+[//]: # (Create the issue)
+[//]: # (Pin the issue on GitHub)
+
+## 😶‍🌫 Meta
+* Scope: Node|Miner
+* Is this linked with a netwok upgrade, and thus mandatory? Yes|No
+* Related network upgrade version: n/a|nvXX
 
 ## 🚢 Estimated shipping date
 
-<Date this release will ship on if everything goes to plan (week beginning...)>
-
-## ✅ Release Checklist
-
-**Note for whoever is owning the release:** please capture notes as comments in this issue for anything you noticed that could be improved for future releases.  There is a *Post Release* step below for incorporating changes back into the [RELEASE_ISSUE_TEMPLATE](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md), and this is easier done by collecting notes from along the way rather than just thinking about it at the end.
+[//]: # (If/when we know an exact date, remove the "week of".)
+[//]: # (If a date week is an estimate, annotate with "estimate".)
 
-- [ ] Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release related changes to this branch. If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.
-- [ ] Bump the version in `build/version.go` in the `master` branch to `vX.Y.(Z+1)-dev` (bump from feature release) or `vX.(Y+1).0-dev` (bump from mandatory release). Run make gen and make docsgen-cli before committing changes
-- [ ] *Optional:* If preparing a release for a network upgrade, make sure all [Lotus dependecies are updated to the correct versions for the network upgrade](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/Update_Dependencies_Lotus.md)
+| Candidate | Date | Release URL |
+|-----------|------|-------------|
+| RC1 | Week of YYYY-MM-DD | |
+| Stable (non-RC) | Week of YYYY-MM-DD (estimate) | |
 
-**Prepping an RC**:
+## 🪢 Dependencies for releases
+> [!NOTE]
+> 1. This is the set of changes that need to make it in for a given RC.  This is effectively the set of changes to cherry-pick from master.  
+> 2. They can be checked as done once they land in `master`.  
+> 3. They are presented here for quick reference, but backporting is tracked in each `Release Checklist`.
```

Comment:
I went back and forth about tracking these here vs. adding them to each RC section below in Release Chcklist.  I figured it was useful to see what changes we are staging for which RC in a consolidated place like we did https://github.com/filecoin-project/lotus/issues/12109 .  I am also fine to inline it.

Reply:
Per 2024-08-09 verbal, we're going to go with this approach for now and can adjust as we do a 1.29.0 release.